### ISSUE TITLE
Enable navigation bar

### ISSUE THREAD

**Issue by jor1196** [ubports/anbox#29]
I would like the "navigation bar" to be enabled, most of all the "back" and "menu" buttons (como en Sailfish OS), for old and new applications that still use them and are not handled with gestures

**Comment by tv0ll** [comment]
I agree. Some apps you just get locked in and can't move out. For example, in F-Droid, if I start looking at screenshots in fullscreen, the only way to back out is with the back arrow on the navigation bar. Because of this, I get stuck and need to close and re-open that app.

**Comment by ITwrx** [comment]
yes, it's very tedious to set up an android app with every settings "page" having to have the whole app restarted b/c you can't go back to previous page. In some cases i can imagine it would make setting something impossible if you couldn't save the previous screen b/c you had to set some sub setting.
